ci/ci.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+set -e
+CI_DIR="/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/ci/"
+
+install_go_kubectl_kind(){
+wget -qO- https://golang.org/dl/go1.20.1.linux-amd64.tar.gz |tar -C /home/runner -xz
+export PATH="${PATH}:/home/runner/go/bin"
+echo "export PATH=\"${PATH}:/home/runner/go/bin/:home/runner/.local/bin/\"" >>~/.bashrc
+go install sigs.k8s.io/kind@v0.20.0
+
+wget -q https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl -O /home/runner/go/bin/kubectl 
+chmod +x /home/runner/go/bin/kubectl
+}
+
+
+create_kind_cluster(){
+kind create cluster
+kubectl get all --all-namespaces
+
+#docker run -itd --device=/dev/hugepages:/dev/hugepages --privileged -v "$(pwd)/docker/vpp-centos-userspace-cni/:/etc/vpp/" --name vpp ligato/vpp-base
+sleep 10
+cd /runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/
+
+docker build . -f ./docker/userspacecni/Dockerfile -t userspacecni:latest
+kind load docker-image userspacecni
+docker exec -i kind-control-plane bash -c "mkdir -p '/runner/_work/userspace-cni-network-plugin/'"
+docker cp "/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/" "kind-control-plane:/runner/_work/userspace-cni-network-plugin/"
+}
+
+deploy_multus(){
+## Multus main branch has major bugs so we fix version
+cd $CI_DIR
+MULTUS_VERSION="v4.0.2"
+wget https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/$MULTUS_VERSION/deployments/multus-daemonset.yml
+sed -i "s/snapshot-thick/v4.0.2/g" multus-daemonset.yml
+kubectl apply -f ./multus-daemonset.yml
+}
+
+vpp_e2e_test(){
+cd $CI_DIR/vpp_test_setup/
+echo "Setting up vpp host"
+./vpp_host.sh
+sleep 20
+echo "Setting up vpp pods"
+./vpp_pod.sh
+
+sleep 20
+kubectl get all -A
+
+kubectl exec -n vpp vpp-app1-kind-control-plane -- ./vpp_pod_setup_memif.sh
+kubectl exec -n vpp vpp-app2-kind-control-plane -- ./vpp_pod_setup_memif.sh
+kubectl exec -n vpp vpp-app1-kind-control-plane -- vppctl "sh int address"
+kubectl exec -n vpp vpp-app2-kind-control-plane -- vppctl "sh int address"
+
+kubectl exec -n vpp vpp-app1-kind-control-plane -- vppctl "ping  192.168.1.4"
```

Comment:
are these IP's hardcoded set ?
just wondering if the pods could end up with different IP's

Reply:
all ip's are static